### DISCUSSION TITLE
password to access my webbox

### DISCUSSION THREAD

**Discussion by robertp007** [jpcornil-git/HA-sma#9] [Q&A]
Hello, 
the password is required to access my webbox. How can I configure the password in the configuration.yaml?
Thanks

**Comment by jpcornil-git** [comment]
Hi,
You need a password to access the UI but not for the RPC API afaik.
Did you try the integration itself and it failed or tested with the sma_webbox.py as explained at the bottom of the readme/main page ? 
Cheers,
jpc

**Comment by jpcornil-git** [comment]
Ending the discussion as there is no further feedback
